include/vcpkg/base/messages.h

```diff
@@ -407,6 +413,7 @@ namespace vcpkg
                     (msg::command_line),
                     "",
                     "'{command_line}' can only add one artifact at a time.");
+    DECLARE_MESSAGE(AddCommandFirstArg, (), "", "The first parameter to add must be 'artifact' or 'port'.");
```

Comment:
This message is command-specific and should probably be in that command's cpp

Reply:
The current design puts *all* messages here, there are no longer any messages in specific .cpps.